package.json

```diff
@@ -81,6 +81,7 @@
     "ember-load-initializers": "^2.1.2",
     "ember-maybe-import-regenerator": "^0.1.6",
     "ember-metrics": "1.2.0",
+    "ember-mockdate-shim": "^0.1.0",
```

Comment:
I can't figure out if this adds anything interesting on top of https://github.com/boblauer/MockDate which is probably directly usable.

Reply:
i don't think it does, but i like the freeze/unfreeze helpers.

Reply:
🆗 🆒